src/style.css

```diff
@@ -1 +1,124 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
You're using the '*' selector here. This is generally not recommended because it can impact performance. It would be better to apply these styles to the specific elements that need them.